src/DynamoCoreWpf/Views/Preview/Watch3DView.xaml.cs

```diff
@@ -461,6 +476,8 @@ private void SetupScene()
                 Position = new Point3D(10, 15, 10),
                 LookDirection = new Vector3D(-10, -10, -10),
                 UpDirection = new Vector3D(0, 1, 0),
+                NearPlaneDistance = 1,
+                FarPlaneDistance = 2000000,
```

Comment:
Not without messing with the internals of the ViewProjectionMatrix. There are tricks to have an "infinite" far plane in DirectX, but these come with downsides. For instance, Z buffer precision has a huge falloff. And there's a lot of numerical instability when the near clip plane is really small.